### PR TITLE
Fix display issue with long app names on Welcome

### PR DESCRIPTION
**Pull request by devinjameson**
Screen Shot 2020-10-05 at 12 04 27 PM
Screen Shot 2020-10-05 at 12 04 47 PM

### CODE REVIEW

File: src/Welcome.tsx [hunk 1]
```diff
@@ -44,56 +44,65 @@ const Welcome: FunctionComponent = () => {
...
   return (
     <>
       <StatusBar backgroundColor={Colors.primaryLightBackground} />
       <ScrollView
-        alwaysBounceVertical={false}
+        style={style.container}
         contentContainerStyle={style.contentContainer}
+        alwaysBounceVertical={false}
       >
-        <GradientBackground gradient={Colors.gradient10} angleCenterY={0.75}>
```
Comment (devinjameson) [inline src/Welcome.tsx:54]: Yeet.

**Review by aledustet — APPROVED** [review]
🚢  :shipit:

File: src/Welcome.tsx (hunk 1 above)
Comment (aledustet) [reply src/Welcome.tsx:54]: Yeet indeed

Comment (johnschoeman) [reply src/Welcome.tsx:54]: Yeet

**Review by johnschoeman — APPROVED** [review]
🚢

File: src/Welcome.tsx (hunk 1 above)
Comment (aledustet) [reply src/Welcome.tsx:54]: Emoji idea, a `yeeti` which is a Yeti with a shirt on that says `YEETI`

**Pull request merged**